lib/IRGen/IRGenModule.cpp

```diff
@@ -367,17 +367,39 @@ IRGenModule::IRGenModule(IRGenerator &irgen,
   // TODO: Check that the deployment target supports the new
   // calling convention. Older versions of the runtime library
   // may not contain the entries using the new calling convention.
-
+  //
   // Only use the new calling conventions on platforms that support it.
+  //
+  // NB: Runtime is built with target clang, swiftc (this file) is
```

Comment:
The runtime is not built with the target clang. That would break cross-compiling. It's built with a just-built host clang as opposed to the build system's clang.

(host-build-target)